### ISSUE TITLE
Fences are always placed as multiparts

### ISSUE THREAD

**Issue by coderbot16** [Juuxel/VanillaParts#13]
Even when just placing down a fence in an empty block space, it's placed as a multipart container. This should only happen when something already exists in the block space!

**Comment by Loriborn** [comment]
This is the same issue I had mentioned in #8 and if fixed would make #11 also unnecessary as regularly placed fences (ie. not multiblock) supporting leads would be expected behavior.

**Comment by Juuxel** [comment]
Yeah, the fence placement code needs some cleanup. #11 is still something I'd like to add at some point, though.
